source/features/show-names.tsx

```diff
@@ -16,7 +16,8 @@ const batchUpdateLinks = batchedFunction(async (batchedUsernameElements: HTMLAnc
 	const myUsername = getUsername();
 	for (const element of new Set(batchedUsernameElements)) {
 		const username = element.textContent;
-		if (username && username !== myUsername && username !== 'ghost') {
+
+		if (username && username !== myUsername && username !== 'ghost' && element.href.endsWith(`/${username}`)) {
```

Comment:
When does this not end with /username?

Reply:
You can see on my description.

In my case the element's textContent is '김정현' and href is 'ki.jh11012'.
'김정현' is user's nickname not github username. So in this feature, no need to query its nickname since it is already showing.

Reply:
If the selector is selecting the wrong thing, maybe we should alter it. Is there any simple change that would avoid this selection? For example by appending `:not(.followed *)` if that class exists on a parent

Reply:
I'll digging is there some way to specifying these with css selector.

Reply:
I think there is no way to detect `followed` article just with css selector.

Reply:
`:not([aria-label="card content"] *)`

Reply:
This href-based condition fails when people set the same name, for example (no RGH enabled)

<img width="354" alt="Screenshot" src="https://github.com/refined-github/refined-github/assets/1402241/392f43bd-5a45-4764-9c8f-36bc3b6e066c">

Reply:
- It have been tried to excluding card content before on this issue.
  - https://github.com/refined-github/refined-github/issues/6530
- Since `:not([aria-label="card content"] *)` can avoid above case also, just using it and discarded old selector: `:not(.color-shadow-medium a)`

Reply:
Thanks for the change! I think we can drop this exclusion though, I'd rather tweak the selector over time if necessary

Reply:
Oops, sorry. I'll drop it